modules/@apostrophecms/piece-type/index.js

```diff
@@ -1156,19 +1157,29 @@ module.exports = {
         });
       },
       getManagerApiProjection(req) {
-        if (!self.options.managerApiProjection) {
-          return null;
-        }
-
-        const projection = { ...self.options.managerApiProjection };
+        // Start from the configured projection, or fall back
```

Comment:
If `managerApiProjection` is not configured at all, then we already bail out and don't project at all, the default behavior is and will remain "fetch everything" (see above).

So what I had in mind was different. We should *always* fetch these fields even if they are *not* included in the projection. In the same spirit as including the columns even if they are not part of the projection: we know you need them even if you don't.

We could reinforce this by accepting and documenting the value `true` ("please fetch less stuff according to your best guess, I have no fields to add") as an alternative to turning it on with an empty `{}`, although the latter should have the same effect ("please use your best guess, plus these fields... I guess I don't have any to suggest").